### PR TITLE
displayPage: Switch to AdwSpinRow

### PR DESCRIPTION
**Pull request by TitouanReal**
Replace the combination of AdwActionRow + GtkSpinButton with an AdwSpinRow.

This is the first time I write Javascript, and the first time I contribute to an extension, so I hope I did it right. Let me know if I need to improve something.

### CODE REVIEW

**Comment by stuarthayhurst** [comment]
Working on my system, looks good to me, thanks :)

**Review by stuarthayhurst — APPROVED** [review]

File: caffeine@patapon.info/preferences/displayPage.js [hunk 1]
```diff
@@ -90,23 +92,13 @@ class CaffeineDisplayPage extends Adw.PreferencesPage {
...
-            numeric: true,
-            valign: Gtk.Align.CENTER
-        });
-        let posIndicatorOffsetRow = new Adw.ActionRow({
-            title: _('Status indicator position'),
-            subtitle: _('The position relative of indicator icon to other items'),
-            activatable_widget: this.posIndicatorOffsetButton
-        });
-        posIndicatorOffsetRow.add_suffix(this.posIndicatorOffsetButton);
+        })
```
Comment (stuarthayhurst) [inline caffeine@patapon.info/preferences/displayPage.js:95]:
```suggestion
        });
```

Missing a semi-colon

Comment (TitouanReal) [reply caffeine@patapon.info/preferences/displayPage.js]: It's fixed now. Thank you

**Review by pakaoraki — APPROVED** [review]

**Comment by pakaoraki** [comment]
Thanks

**Comment by stuarthayhurst** [comment]
Are we waiting for the pipeline fix before merging this? Otherwise I'm happy to merge it :)

**Pull request merged**